sunpy/map/mapbase.py

```diff
@@ -401,7 +401,7 @@ def _repr_html_(self, compute_dask=False):
         ax.semilogy()
         # Explicitly set the power limits for the X axis formatter to avoid text overlaps
         ax.xaxis.get_major_formatter().set_powerlimits((-3, 4))
-        ax.set_xlabel('Pixel value in linear bins')
+        ax.set_xlabel(f'Pixel value{" (" + str(self.unit) +") " if self.unit is not None else " "}in linear bins')
```

Comment:
Can we not tidy this string up a bit?

Reply:
Was thinking to create an temp variable:
```
display_unit = f"({str(self.unit)})" if self.unit is not None else " "
ax.set_xlabel('Pixel value' + display_unit + 'in linear bins') 
```

Reply:
That sounds worse?

Reply:
This looks better:
```
display_unit = f' ({self.unit})' if self.unit is not None else ''
ax.set_xlabel(f'Pixel value{unit_string} in linear bins')
```

Reply:
I don't think we need a variable for this.

Reply:
Shorten the code: 

ax.set_xlabel(f"Pixel value{' (' + self.unit + ')' if self.unit else ''} in linear bins")

Reply:
Sure

Reply:
Updated the code